src/assets/lang/ru.json

```diff
@@ -179,8 +179,8 @@
-      "coronavirus1": "**Нам жаль, что вы плохо себя чувствуете. К сожалению, симптомы, которые вы указали сегодня, могут быть признаками COVID-19.**\n\nПомните, что при наличии легких симптомов большинство людей могут выздороветь дома без медицинской помощи.\n\nОднако мы рекомендуем любому человеку с симптомами COVID-19 для защиты себя и окружающих предпринять следующее:\n\n**Оставайтесь дома.** Избегайте контактов с другими людьми (даже с родственниками), чтобы не заразить их.\n\n**Обратитесь к поставщику медицинских услуг.** Позвоните своему врачу или на горячую линию штата Нью-Йорк по вопросам здоровья по телефону [1-888-364-3065](тел:1234).\n\n**Сдайте анализ.** Поговорите со своим поставщиком медицинских услуг или запишитесь на прием в любом общедоступном пункте тестирования. \n\n[Найти ближайший пункт.](https://coronavirus.health.ny.gov/find-test-site-near-you)",
+      "noSymptomsNotFeelingWell": "**Спасибо, что заполнили журнал сегодня. Нам жаль, что вы плохо себя чувствуете.**\n\nК счастью, вы сообщили лишь о нескольких симптомах. Если ваше самочувствие не улучшится или появятся новые симптомы, поговорите со своим поставщиком медицинских услуг или позвоните на горячую линию штата Нью-Йорк по вопросам здоровья по телефону [1-888-364-3065](tel:18883643065).\n\nЕсли вы хотите сдать анализ, обратитесь к своему поставщику медицинских услуг или запишитесь на прием в любом общедоступном пункте тестирования. \nНайти ближайший пункт.\n\nЖдем вас завтра, чтобы узнать о вашем самочувствии.",
+      "coronavirus1": "**Нам жаль, что вы плохо себя чувствуете. К сожалению, симптомы, которые вы указали сегодня, могут быть признаками COVID-19.**\n\nПомните, что при наличии легких симптомов большинство людей могут выздороветь дома без медицинской помощи.\n\nОднако мы рекомендуем любому человеку с симптомами COVID-19 для защиты себя и окружающих предпринять следующее:\n\n**Оставайтесь дома.** Избегайте контактов с другими людьми (даже с родственниками), чтобы не заразить их.\n\n**Обратитесь к поставщику медицинских услуг.** Позвоните своему врачу или на горячую линию штата Нью-Йорк по вопросам здоровья по телефону [1-888-364-3065](tel:18883643065).\n\n**Сдайте анализ.** Поговорите со своим поставщиком медицинских услуг или запишитесь на прием в любом общедоступном пункте тестирования. \n\n[Найти ближайший пункт.](https://coronavirus.health.ny.gov/find-test-site-near-you)",
```

Comment:
We have to admire the thoroughness of the translators here, as well as translating `tel` they replaced the `(` and `)` with the wider Chinese variants.

We should also ask them not to in this particular case 🙂 (we should prob check the other Markdown links too)